core/genesis.go

```diff
@@ -259,9 +259,6 @@ func (g *Genesis) ToBlock(db ethdb.Database) *types.Block {
 		ShardStateHash: g.ShardStateHash,
 		ShardState:     shardStateBytes,
 	}
-	if g.GasLimit == 0 {
```

Comment:
what is the default gas limit now?

Reply:
Doesn't matter, as we don't process tx now.